Client/Application/WebServer.swift

```diff
@@ -18,8 +18,10 @@ class WebServer {
     }
 
     func start() -> Bool {
-        if !server.running {
-            try! server.startWithOptions([GCDWebServerOption_Port: 6571, GCDWebServerOption_BindToLocalhost: true, GCDWebServerOption_AutomaticallySuspendInBackground: true])
+        withExtendedLifetime(server) {
```

Comment:
I wonder if changing the local class property server in WebServer to a static declaration would force it to stay alive? We could also replace the sharedInstance class method with the static keyword but I don't know if that would make any difference since static is the same as 'class final'.